Mathlib/Data/List/Nodup.lean

```diff
@@ -3,9 +3,9 @@ Copyright (c) 2018 Mario Carneiro. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro, Kenny Lau
 -/
+import Mathlib.Data.Fin.Basic
```

Comment:
Why do you need this import? The init file that was imported before only has deprecated stuff in it. So this file should be able to work with the core versions, right?

Reply:
Fixed.